src/PrestaShopBundle/Service/Hook/Finder.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+namespace PrestaShopBundle\Service\Hook;
+
+/**
+ * This class declares the functions needed to get structured data
+ * from the modules, by asking them to follow a specific class.
+ */
+class Finder {
```

Comment:
Commit message says the class is abstract. Can you use the class as is, or you forgot to make it abstract?

Also about the same, I'm not very confortable with Finder since there is a [Symfony component named finder](http://symfony.com/doc/current/components/finder.html), I'm scared it will be confusing. How about `PrestaShopBundle\Service\Hook\HookFinder` ?